index.html

```diff
@@ -7,146 +7,146 @@
-						href : "https://github.com/w3c/wot-architecture/graphs/contributors"
-					} ]
-				}, {
-					key : "Repository",
-					data : [ {
-						value : "We are on GitHub",
-						href : "https://github.com/w3c/wot-architecture/"
-					}, {
-						value : "File a bug",
-						href : "https://github.com/w3c/wot-architecture/issues"
-					} ]
-				} ],
-		localBiblio : {
-			"CoRE-RD" : {
-				href : "https://tools.ietf.org/html/draft-ietf-core-resource-directory-11",
-				title : "CoRE Resource Directory",
-				status : "Internet-Draft",
-				publisher : "IETF",
-				date : "03 July 2017"
-			},
-			"WOT-SECURITY-CONSIDERATIONS" : {
-				href : "https://www.w3.org/TR/wot-security/",
-				title : "Web of Things Security and Privacy Considerations",
-				publisher : "W3C"
-			},
-			"WOT-SECURITY-BEST-PRACTICES" : {
-				href : "https://www.w3.org/TR/wot-security-best-practices/",
-				title : "Web of Things Security Best Practices",
-				publisher : "W3C"
-			},
-			"WOT-SECURITY-TESTING-PLAN" : {
-				href : "https://www.w3.org/TR/wot-security-testing-plan/",
-				title : "Web of Things Security Best Practices",
-				publisher : "W3C"
-			},
-			"IEC-FOTF" : {
-				href : "https://www.iec.ch/whitepaper/pdf/iecWP-futurefactory-LR-en.pdf",
-				title : "Factory of the future",
-				publisher : "IEC",
-				date : "October 2015"
-			},
-			"iot-schema-org" : {
-				href : "https://iot.schema.org/",
-				title : "iot.schema.org"
-			},
-			"REST" : {
-				title : "REST: Architectural Styles and the Design of Network-based Software Architectures",
-				author : "Roy Thomas Fielding",
-				status : "PhD thesis",
-				publisher : "University of California, Irvine",
-				date : "2000"
-			},
-			"SAREF" : {
-				href : "https://sites.google.com/site/smartappliancesproject/ontologies/reference-ontology",
-				title : "Smart Appliances REFerence (SAREF) ontology",
-				publisher : "ETSI",
-				date : "November 2015"
-			},
-			"SSN" : {
-				href : "https://www.w3.org/TR/vocab-ssn/",
-				title : "Semantic Sensor Ontology",
-				publisher : "W3C",
-				date : "October 2017"
-			},
-			"HMI" : {
-				title : "The Encyclopedia of Human-Computer Interaction, 2nd Ed",
-				author : "Mads Soegaard, Rikke Friis Dam",
-				publisher : "Interaction Design Foundation",
-				date : "2013"
-			},
-			"NORMAN" : {
-				title : "The Psychology of Everyday Things",
-				author : "Donald A. Norman",
-				publisher : "Basic Books",
-				date : "1988"
-			},
-			"MQTT" : {
-				title : "MQTT Version 3.1.1",
-				author : "Andrew Banks, Rahul Gupta",
-				publisher : "OASIS Standard",
-				date : "2014"
-			},
-			"FORMS" : {
-				href : "https://tools.ietf.org/html/draft-hartke-t2trg-coral",
-				title : "The Constrained RESTful Application Language (CoRAL)",
-				author : "Klaus Hartke",
-				publisher : "IETF",
-				status : "Internet-Draft",
-				date : "2019"
-			}
-		}
-	};
+        var respecConfig = {
+                specStatus : "ED",
```

Comment:
Nowadays tabs are expanded to 2 or 4 spaces in these documents, but it doesn't matter.

Reply:
Yeah, I may do some cleanup.   At any rate, I got rid of the tabs, and intend to keep them out of the document.  VI set up now; sorry, some of the tabs might have been my fault; new machine, forgot to set up VI "properly".

Reply:
This was causing conflicts so I just reverted this file to the one in the master repo.   We can deal with the tab issue later...